msb/fusionlog/TimeSeriesLogger.py

```diff
@@ -46,7 +46,8 @@ def write(self, data):
             self._update_filehandle(ts)
 
         # sort data values by key order and concatenate as str
-        row = ",".join((str(data[key]) for key in self.key_order)) + "\n"
+        # row = ",".join((str(data[key]) for key in self.key_order)) + "\n"
+        row = f'{",".join(map(str, [data[key] for key in data.keys()]))}\n'
```

Comment:
data.keys is a set. Therefore the order if the keys will not be stable.
Also this should not be done in this branch.